### ISSUE TITLE
Use of depreceated sonstants for humidity and temperature

### ISSUE THREAD

**Issue by TiberiusRegenschein** [z8i/ha_custom_klimaloggpro#6]
Home Assistant version 2024.2.0
Server: Intel NUC running debian 11

Error messages by home assistant:

- WARNING (MainThread) [homeassistant.const] TEMP_CELSIUS was used from klimaloggpro, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTemperature.CELSIUS instead, please report it to the author of the 'klimaloggpro' custom integration
- WARNING (MainThread) [homeassistant.const] DEVICE_CLASS_HUMIDITY was used from klimaloggpro, this is a deprecated constant which will be removed in HA Core 2025.1. Use SensorDeviceClass.HUMIDITY instead, please report it to the author of the 'klimaloggpro' custom integration

**Comment by dstueven** [comment]
Unfortunately I see the same messages in my log. 
And I never got any data into the dashboard nor do I get the Klimalog Pro main station to establish the USB connection (timing out).
I tried it on two installations:
1) On a synology NAS 218+ in Virtual station mapping the USB key as a USB2 device into the VM (VM is HAOS).
2) A HAOS standard install on a Rasberry 3B.
I do see the USB key appear in the operating system and see it listed when using the ls command.
@TiberiusRegenschein : Where you able to get the sync work?

**Comment by TiberiusRegenschein** [comment]
@dstueven , I got the KlimalogPro with no issue to work in a docker installation after I followed the advice in "Attention: USB Device!". and added the user to the "plugdev" group.

A typical issue of USB devices used on USB 3.0 ports is that there is interference. I decided to keep the USB stick connected with an USB extension cable (about 30 cm long and keep it as far away from any USB 3.0 port as possible => In your case it may not be a software issue but a physics issue due to interference

**Comment by dstueven** [comment]
@TiberiusRegenschein : Thank you for your answer. I got it working now. I tested the USB dongle with the windows software and afterwards it also worked with Home Assistant. I think the problem was that the base station was not replicated since it ran out of battery and lost all data. It looks like the base station stores which „Logger“ has been selected when relocating with windows and that „Logger 1“ is assumed by the HA integration. At least it worked right away after I used it with the windows client. Thanks again.
